takwimu/templates/takwimu/_includes/sections/support-services/tabs.html

```diff
@@ -1,18 +1,19 @@
 <section class="support-services-tabs">
   <div class="bg-secondary">
     <nav
-        class="nav nav-justified container text-center"
+        class="nav nav-justified container text-center custom-nav"
         id="nav-tabs" role="tablist">
       {% for service in support_services %}
-        <a class="nav-item nav-link text-light p-0 pt-4"
-          id="support-{{ service.get_slug | add:'-btn' }}"
-          data-toggle="tab" href="#support-{{ service.get_slug }}" role="tab"
-          aria-controls="support-{{ service.get_slug }}"
-          aria-selected="true">
+      <a class="nav-item nav-link text-light pt-3 px-0"
+        id="support-{{ service.get_slug | add:'-btn' }}"
+        data-toggle="tab" href="#support-{{ service.get_slug }}" role="tab"
+        aria-controls="support-{{ service.get_slug }}"
+        aria-selected="true">
+        <div class="text-center {% if not forloop.first %}border-left{% endif %}">
           <h2 class="mb-1">{{ service.icon.as_html }}</h2>
-          <p class="small text-uppercase ">{{ service.title }}</p>
-          <div class="invisible bg-white" style="height:5px;"></div>
```

Comment:
Are we removing the invisible div and replacing it with a `bottom border`? Is this a better approach?

Reply:
@KhadijaMahanga 
It's best since it removes one entire html element with two css class by one simple css line